spec/requests/api/v1/participant_declarations_spec.rb

```diff
@@ -108,6 +108,43 @@ def build_params(attributes)
         end
       end
 
+      context "when declaration is rejected" do
+        let(:valid_params) do
+          {
+            participant_id: ect_profile.user.id,
+            declaration_type: "started",
+            declaration_date: (ect_profile.schedule.milestones.first.start_date + 2.days).rfc3339,
+            course_identifier: "ecf-induction",
+          }
+        end
```

Comment:
I don't think `when declaration is rejected` is a meaningful context here. That statement is the outsome, so it better belongs in the `it` statements.

Also, why do we need different `valid_params` here? Do we actually need this context?